scripts/node/dictionary/formats.js

```diff
@@ -104,7 +108,14 @@ const formatJsManifest = (dictionary) => {
     motions: filterByCategory(otherProperties, 'motion'),
   };
 
-  return `module.exports = ${JSON.stringify(result, null, 2)}`;
+  Object.entries(result).map((name, value) => console.log(name));
```

Comment:
[could] Do we still need this console.log in here? Or maybe it's an added benefit to see this stuff logged out?